### ISSUE TITLE
Did ASRC2020  have papers?

### ISSUE THREAD

**Issue by cjgdo** [espnet/espnet#4624]
![image](https://user-images.githubusercontent.com/61341292/188884782-bec6a6d2-a3b4-49ec-8173-a51c3ca57224.png)

i want to know asrc2020 have some papers?

i have see the conf/encoder  、decoder, but i did not see some new contribution  in  there

**Comment by b-flo** [comment]
Hi,

> Did ASRC2020 have papers? 
> i want to know asrc2020 have some papers?

I think you mean A**E**SRC2020!
If you're asking about the challenge paper, see: https://arxiv.org/pdf/2102.10233.pdf.
If you're asking about a paper talking about the ESPnet system/recipe, I'm not aware of any. I'll let other answer.

**Comment by sw005320** [comment]
> If you're asking about a paper talking about the ESPnet system/recipe, I'm not aware of any. I'll let other answer.

Yes, we just released this recipe, but we do not have any paper describing the recipe.

**Comment by cjgdo** [comment]
ok,thank you for your answer
